allensdk/brain_observatory/behavior/eye_tracking_processing.py

```diff
@@ -207,10 +207,16 @@ def process_eye_tracking_data(eye_data: pd.DataFrame,
                                             outliers,
                                             dilation_frames=dilation_frames)
 
+    # remove outliers/likely blinks from default `pupil_area` column
+    # maintain full fits in the `pupil_area_raw` column
+    pupil_area_raw = pupil_areas.copy()
+    pupil_areas[likely_blinks == True] = np.nan
```

Comment:
I think there needs to be one more line changed:

https://github.com/AllenInstitute/AllenSDK/blob/a0d7e720c49062c51f7fc4c090f65daf84ab8250/allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_nwb_api.py#L513

where `area=eye_tracking_df['pupil_area'].values,` should be changed to `area=eye_tracking_df['pupil_area_raw'].values,`.

Reply:
@njmei maybe i am misunderstanding your suggestion here, but the desire is to have both `eye_tracking_df['pupil_area_raw']` and `eye_tracking_df['pupil_area'] `with `eye_tracking_df['pupil_area']` being the one that is filtered with likely blinks removed. i think both of those need to be in NWB also.

Reply:
@matchings I see... in that case then the NWB extension will also need to be updated and the extension rebuilt if that data also needs to live in the NWB file:

https://github.com/AllenInstitute/AllenSDK/blob/rc/2.7.0/allensdk/brain_observatory/nwb/eye_tracking/extension_builder.py

Based on Doug's PR though, it seems like his intent was to save only the `pupil_area_raw` and regenerate the `pupil_area` column when loading from NWB/LIMS APIs.

Reply:
ok, i guess that is fine (regenerating the pupil area column when loading with SDK but not saving it that way in NWB), as long as we make it clear to users who dont use the SDK and only rely on NWBs that the pupil_area_raw needs to have the blinks filtered out. It would be better to do this for them, since that is what nearly everyone will want, but if it requires rebuilding things, its probably not worth the extra effort right now.